project/plugins.sbt

```diff
@@ -8,4 +8,4 @@ addSbtPlugin("org.scoverage"      % "sbt-scoverage"          % "1.5.1")
 addSbtPlugin("org.xerial.sbt"     % "sbt-sonatype"           % "2.3")
 
 addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject"      % "0.6.0")
-addSbtPlugin("org.scala-js"       % "sbt-scalajs"                   % "0.6.25")
+addSbtPlugin("org.scala-js"       % "sbt-scalajs"                   % "0.6.24")
```

Comment:
Why the downgrade here?

Reply:
apparently 0.6.25 works with Scala 2.13.0-M5 but not 2.13.0-M4, see 
https://github.com/typelevel/cats-collections/pull/130#issuecomment-427152636

it's plausible, there's no real reason for Seb to support multiple 2.13 milestones in the same version

Reply:
Yes, there's indeed a bug that prevents Scalacheck from being used with M4. But our upstream dependencies are not ready for M5 yet. So that's the only compromise I could come up with.

Reply:
👍